components-api/src/test/scala/pl/touk/nussknacker/engine/api/typed/SubclassDeterminerSpec.scala

```diff
@@ -0,0 +1,39 @@
+package pl.touk.nussknacker.engine.api.typed
+
+import org.apache.commons.lang3.ClassUtils
+import org.scalatest.funsuite.AnyFunSuite
+import org.scalatest.matchers.should.Matchers
+
+class SubclassDeterminerSpec extends AnyFunSuite with Matchers {
+
+  test("Should validate assignability for decimal types") {
```

Comment:
It would be great to have a precise comparison of our 2 ways of comparing types (currently from StrictConversionDeterminer and ImplcitConversionDeterminer). I'd prefer to do it on a one test data table and use it in 2 parameterized TableDrivenPropertyChecks tests. Then we have a proof that the strict checker work on other data types like maps, lists, records, tagged types.

Reply:
I have added such tests in AssignabilityDeterminerSpec. I only compare the numerical types that actually differ in those two Determiners. The rest is already tested in typingResultSpec.